android/src/test/java/io/ably/lib/test/common/android/AssetResourceLoader.java

```diff
@@ -0,0 +1,31 @@
+package io.ably.lib.test.common.android;
```

Comment:
So this is now identical to the JRE resource loader apart from the path for reading the files - should we instead refactor it so there is just a single common class?